IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -107,6 +107,13 @@ - (BOOL)nsLoggingEnabled
     return _nsLoggingEnabled;
 }
 
+- (BOOL)euiiMaskingEnabled
```

Comment:
nit: Should we use an enum instead of euiiMaskingEnabled and piiLoggingEnabled flags? Might be hard to implement (since it is used in public API), but it will be more elegant  -- as you mentioned right now `euiiMaskingEnabled ` is ignored when `piiLoggingEnabled ` is false.

Reply:
Agree, enum would be better here if we will need to enable masking for additional types. I'll see how difficult it would be to make a change.

Reply:
@antrix1989 - addressed, would appreciate another look.